src/sklearn_raster/features.py

```diff
@@ -84,16 +84,82 @@ def apply_ufunc_across_features(
         allow_cast: bool = False,
         check_output_for_nodata: bool = True,
         keep_attrs: bool = False,
+        inner_thread_limit: int | None = 1,
         **ufunc_kwargs,
     ) -> FeatureArrayType | tuple[FeatureArrayType]:
-        """Apply a universal function to all features of the array."""
+        """
+        Apply a universal function to all features of the array.
+
+        Parameters
+        ----------
+        func : callable
+            The universal function to apply across features. The function should accept
+            an array of shape (samples, features) and return one or more arrays of shape
+            (samples, size), where sizes is defined by output_sizes.
```

Comment:
Should this be "where *size* is defined by output_sizes" to match "(samples, size)"